packages/components/src/components/tag/_mixins.scss

```diff
@@ -11,6 +11,7 @@
   color: $text-color;
   background-color: $bg-color;
```

Comment:
Should we add `transition: background-color $duration--fast-01 motion(entrance, productive);`?

Reply:
Oh yeah that would be nice. I always forget about motion.